### PR TITLE
Autumn 2019 Mk. 2

### PR DESCRIPTION
**Pull request by BenWoodford76**
Re-committing to cut down on the stream of commit messages

Currently a WIP, please do not merge until I'm happy

### CODE REVIEW

**Comment by BenWoodford76** [comment]
Happy to merge, @samozzy could you give it a review please?

File: _shows/19_20/still_alice.md [hunk 1]
```diff
@@ -79,16 +100,18 @@ crew:
...
 prod_shots: x24nSs
 
 assets:
   - type: poster
     image: fmpWm9s
-
-# Programme - https://drive.google.com/file/u/3/d/1PgJRxNl79t-asv27p20gZBAvpYfhRJJD/preview
+  - type: programme
+    title: Still Alice Programme
+    filename: still-alice-programme.pdf
```
Comment (samozzy) [inline _shows/19_20/still_alice.md]: This 404s

Comment (BenWoodford76) [reply _shows/19_20/still_alice.md]: it's in the LFS and references the correct filename... :(

File: _shows/18_19/franz_and_marie_edinburgh.md [hunk 2]
```diff
@@ -32,9 +32,9 @@ crew:
 - role: Publicity Assistant
   name: Alice Walker  
 
-# assets:
-#   - type: poster
-#     image: 
+ assets:
+   - type: poster
+     image: ZSNh3W6
```
Comment (davidtaylorhq) [inline _shows/18_19/franz_and_marie_edinburgh.md]: Less whitespace required
```suggestion
assets:
- type: poster
  image: ZSNh3W6
```

File: _shows/19_20/order_from_chaos_lakeside.md [hunk 3]
```diff
@@ -28,9 +28,9 @@ crew:
   name: Rachel Elphick
 
 
-# assets:
-# - type: poster
-#   image: r85cqFg
+ assets:
+ - type: poster
+   image: r85cqFg
```
Comment (davidtaylorhq) [inline _shows/19_20/order_from_chaos_lakeside.md]:
```suggestion
assets:
- type: poster
  image: r85cqFg
```

File: _shows/19_20/midsummer.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+adaptor: Tara Anegada
+period: Autumn
+season: In House
+season_sort: 60
+date_start: 2019-11-20
+date_end: 2019-11-23
+venue: New Theatre
+
+# trivia: 
+
```
Comment (davidtaylorhq) [inline _shows/19_20/midsummer.md]:
```suggestion
cast:
```

File: _shows/19_20/midsummer.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+  name: Emi Thackray
+- role: Stagehand
+  name: Alice Clothier
+
+assets:
+  - type: poster
+    image: 8j856N4
+  - type: programme
+    title: A Midsummer Night's Dream Programme
+    filename: Midsummer-2019-Programme.pdf
```
Comment (davidtaylorhq) [inline _shows/19_20/midsummer.md]:
```suggestion
assets:
- type: poster
  image: 8j856N4
- type: programme
  title: A Midsummer Night's Dream Programme
  filename: Midsummer-2019-Programme.pdf
```

**Comment by samozzy** [comment]
Programmes are still angry
Order From Chaos Edinburgh is unhappy

File: _shows/18_19/order_from_chaos_edinburgh.md [hunk 6]
```diff
@@ -29,9 +29,9 @@ crew:
   name: Rachel Elphick
 
 
-# assets:
-# - type: poster
-#   image: r85cqFg
+ assets:
+ - type: poster
+   image: r85cqFg
```
Comment (davidtaylorhq) [inline _shows/18_19/order_from_chaos_edinburgh.md]:
```suggestion
assets:
- type: poster
  image: r85cqFg
```

**Pull request merged**